src/app/Main/Body/InfoPage/Amrs/InfoPage-AmrsContent.component.tsx

```diff
@@ -61,14 +39,17 @@ export function InfoPageAmrsContentComponent(props: {
             {props.amrKeys.map((amrKey) => (
                 <div key={`${amrKey}-amr-table-dialog`}>
                     <p>{t(`Methods.Amrs.${amrKey}.Paragraph`)}</p>
-                    <Button css={tableDialogButtonStyle} onClick={() => handleOpen(amrKey)} color="primary">
+                    <Button
+                        css={tableDialogButtonStyle}
+                        onClick={() => handleOpen(amrKey)}
+                        color="primary"
+                    >
                         {props.tableData[amrKey].title}
                     </Button>
                     <InfoPageAmrDialogComponent
                         resistancesTableData={props.tableData[amrKey]}
-                        isOpen={
-                            infoPageTableDialogsAreOpen[amrKey]
-                        }
+                        amrKey={amrKey}

```

Comment:
Reject: You misunderstood that part of my suggestion: So you still have multiple dialogs defined, i aimed for having only one dialog living in the html dom. So you actually just have to move the dialog out of the armKeys.map, so it lives besides the amr buttons. Then you do not need this new armKey, openArmDialogKey mechanism.

Reply:
Done. Commit "Remove key type & define dialog outside map"